dashboard/app/views/shared/_course_wide_block.haml

```diff
@@ -100,7 +101,18 @@
         .smalltext= body
       = render partial: 'shared/courseblock_action', locals: { id: id, showState: showState, url: url, current_user: current_user }
 
--elsif [Script::COURSE1_NAME, Script::COURSE2_NAME,Script::COURSE3_NAME, Script::COURSE4_NAME, Script::ALGEBRA_NAME].include? id
+-elsif [Script::EXPRESS_NAME, Script::PRE_READER_EXPRESS_NAME].include? id
+  - height = 120
+  .courseblock-span6.courseblock-wide-small{style: "min-height: #{height}px;"}
+    =link_to link_to script_url(id)  do
+      .courseblock-span2.imgspan
+        %img{src: CDO.code_org_url("/shared/images/courses/fill-150x110/logo_tall_#{id}.jpg")}
+      .courseblock-span4.heading
+        %h3= data_t_suffix('script.name', id, 'title')
+        .smalltext= data_t_suffix('script.name', id, 'description_short')
+      .bottombar.smalltext.audience= data_t_suffix('script.name', id, 'description_audience')
```

Comment:
At what point does it make sense to build this in React instead of adding more haml?